apps/system/js/operator_variant_handler.js

```diff
@@ -291,7 +293,8 @@
           'ril.mms.mmsc': 'mmsc',
           'ril.mms.mmsproxy': 'mmsproxy',
           'ril.mms.mmsport': 'mmsport',
-          'ril.mms.authtype': 'authtype'
+          'ril.mms.authtype': 'authtype',
```

Comment:
add authtype & mtu in https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/operator_variant_handler_test.js#L87